android/Omnivore/app/src/main/java/app/omnivore/omnivore/ui/reader/WebReaderContent.kt

```diff
@@ -80,9 +80,9 @@ data class WebReaderContent(
                   id: "${item.savedItemId}",
                   linkId: "${item.savedItemId}",
                   slug: "${item.slug}",
-                  createdAt: new Date(1662571290735.0).toISOString(),
```

Comment:
LOL sorry!

Reply:
I think I used the current days timestamp...as a temporary stub 😬

Reply:
hah no worries, funny it took us so long to notice.